account/repository/user.go

```diff
@@ -153,14 +153,17 @@ func (m *GormUserRepository) Delete(ctx context.Context, id uuid.UUID) error {
 
 	obj := User{ID: id}
 
-	err := m.db.Delete(&obj).Error
+	db := m.db.Delete(&obj)
 
-	if err != nil {
+	if db.Error != nil {
```

Comment:
Is this thread-safe?

Reply:
ah nevermind, I didn't notice that you're assigning to a new `db` variable :)

Reply:
Actually the `db` variable name is a little confusing, could we name it `result` instead, like in the example?

[http://gorm.io/docs/error_handling.html](http://gorm.io/docs/error_handling.html)

Reply:
Yeah.. 'result' would be a better name.

Reply:
Fixed in https://github.com/fabric8-services/fabric8-auth/pull/500/commits/6d5cd7acd5e170920b8f4b35097c72a8fd96bdf5